projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -603,22 +608,56 @@ public function get_all_integrations_status( $request ) {
 		$version      = absint( $request->get_param( 'version' ) );
 		$integrations = array();
 
-		foreach ( array_keys( $this->get_supported_integrations() ) as $slug ) {
-			$plugin_status = $this->get_plugin_status( $slug );
+		foreach ( $this->get_supported_integrations() as $slug => $config ) {
+			$integration_status = ( isset( $config['type'] ) && $config['type'] === 'plugin' )
+				? $this->get_plugin_status( $slug )
+				: $this->get_service_status( $slug );
 
 			if ( 1 === $version ) {
-				$integrations[ $slug ] = $plugin_status;
+				$integrations[ $slug ] = $integration_status;
 			} else {
-				$integrations[] = array_merge(
-					array( 'id' => $slug ),
-					$plugin_status
-				);
+				$integrations[] = array_merge( array( 'id' => $slug ), $integration_status );
 			}
 		}
 
 		return rest_ensure_response( $integrations );
 	}
 
+	/**
+	 * Get status for internal/service integrations.
+	 *
+	 * @param string $slug Service slug.
+	 * @return array Service status data.
+	 */
+	private function get_service_status( $slug ) {
+		switch ( $slug ) {
+			case 'salesforce':
+				return array(
+					'type'        => 'service',
+					'slug'        => 'salesforce',
+					'pluginFile'  => null,
+					'isInstalled' => false,
+					'isActive'    => false,
+					'isConnected' => false,
+					'version'     => null,
+					'settingsUrl' => null,
+					'details'     => array(),
+				);
+			default:
+				return array(
+					'type'        => 'service',
+					'slug'        => $slug,
+					'pluginFile'  => null,
+					'isInstalled' => false,
+					'isActive'    => false,
+					'isConnected' => false,
+					'version'     => null,
+					'settingsUrl' => null,
+					'details'     => array(),
+				);
+		}
+	}
```

Comment:
I set up a generic get_service_status() method to handle non-plugin services like Google, Zapier, Salesforce, etc. The logic to get their status and return them would live here.